edit_course/course_forms.py

```diff
@@ -1,13 +1,36 @@
 from django import forms
+from django.conf import settings
+from django.contrib.auth.models import User
 from django.core.exceptions import ValidationError
 from django.utils.translation import ugettext_lazy as _
 from django_colortag.forms import ColorTagForm
 
+from aplus.api import api_reverse
 from course.models import LearningObjectCategory, CourseModule, CourseInstance, UserTag
 from lib.validators import generate_url_key_validator
 from userprofile.models import UserProfile
 
 
+class SelectManyField(forms.ModelMultipleChoiceField):
```

Comment:
Move the `SelectManyField` class to the file `lib/fields.py`. The class name must be changed since it is not descriptive enough now. Note that the clean method hardcodes references to the User model here. Can the model class be abstracted somehow or is it really necessary to hardcode it?